### PR TITLE
thread suspended when minimized

### PR DESCRIPTION
**Pull request by CedricGuillemet**
0% CPU/GPU usage when minimized

### CODE REVIEW

**Comment by CedricGuillemet** [comment]
Nop! I don't know why it switched to binary

File: Library/Source/RuntimeImpl.cpp [hunk 1]
```diff
@@ -34,7 +38,16 @@ namespace babylon
 
     void RuntimeImpl::Suspend()
     {
-        m_engine->Suspend();
```
Comment (syntheticmagus) [inline Library/Source/RuntimeImpl.cpp]: This seems like something that should be a feature of Arcana, though it doesn't seem to be in there at the moment. I added an issue to Arcana to implement that feature: https://github.com/microsoft/arcana.cpp/issues/4. Can you add a comment or issue (or both) here to remind us to switch to using that feature once it's available?

**Comment by syntheticmagus** [comment]
In reply to the Win32/App.cpp conversation (I can't seem to reply to that directly for some reason), if there are no intentional changes to the file, can you revert whatever Git thinks happened to it? (I think there probably _should_ be changes, but they don't need to be a part of this PR. If we want to properly do suspend/resume in both Win32 and UWP, we can just open an issue to do that in another check-in.)

File: Library/Source/RuntimeImpl.h [hunk 2]
```diff
@@ -50,7 +51,10 @@ namespace babylon
 
         arcana::manual_dispatcher<babylon_dispatcher::work_size> m_dispatcher{};
         arcana::cancellation_source m_cancelSource{};
-        std::mutex m_taskMutex{};
+        std::mutex m_taskMutex;
+        std::mutex m_suspendMutex;
+        std::condition_variable m_suspendVariable;
+        bool m_suspended = false;
```
[the remark this answers is not in the document]
Comment (CedricGuillemet) [reply Library/Source/RuntimeImpl.h]: It makes sense for pointers, base types,... not for objects having default constructor. I did some tests using Godbolt and the code is exactly the same. As it's not necessary, it should be removed. Even return types for lambdas ,for example, seem more necessary than braces here.

[the remark this answers is not in the document]
Comment (bghgary) [reply Library/Source/RuntimeImpl.h]: Necessary or not, it doesn't change the behavior which then makes this a convention question. Let's take this offline and discuss.

**Pull request merged**